### ISSUE TITLE
french citations

### ISSUE THREAD

**Issue by mcfrank** [langcog/wordbank#260]
new unpublished French data (for the cat @kachergis) are from:
Mireille Babineau (Department of Psychology, University of Toronto (St. George)), 
Alex de Carvalho (Université Paris Cité, CNRS, LaPsyDÉ, F-75005 Paris, France)
Anne-Caroline Fievet, (LSCP (ENS, EHESS, CNRS), Département d’Etudes Cognitives, ENS - PSL Research University), 
Cécile Crimon (LSCP (ENS, EHESS, CNRS), Département d’Etudes Cognitives, ENS - PSL Research University; Université Paris Cité), 
Sho Tsuji (International Research Center for Neurointelligence, The University of Tokyo)

I think we just list this as:

Babineau, M., de Carvalho, A., Fievet, A., Crimon, C., and Tsuji, S. (unpublished).

**Comment by HenryMehta** [comment]
@mcfrank I'm not sure to which datasets this refers.  French datasets without a citation are

```
{
        "name": "Kern",
        "dataset": "",
        "instrument_language": "French (French)",
        "instrument_form": "WS",
        "file": "raw_data/French_French_WS/FrenchFrenchWS_Kern.csv",
        "splitcol": false,
        "norming": false,
        "longitudinal": false,
        "contributor": "Sophie Kern, Centre national de la recherche scientifique (CNRS)",
        "license": "CC-BY",
        "citation": ""
    },
    {
        "name": "Bergmann",
        "dataset": "",
        "instrument_language": "French (French)",
        "instrument_form": "WG",
        "file": "raw_data/French_French_WG/FrenchFrenchWG_Bergmann.csv",
        "splitcol": false,
        "norming": false,
        "longitudinal": false,
        "date_format": "%Y-%m-%d",
        "contributor": "Christina Bergmann (Max Planck Institute for Psycholinguistics),\nAnne-Caroline Fievet (Laboratoire de Sciences Cognitives et Psycholinguistique (ENS, EHESS, CNRS), Département d’Etudes Cognitives, Ecole Normale Supérieure, PSL Research University)",
        "license": "CC-BY",
        "citation": "​"
    },
```

**Comment by mcfrank** [comment]
Sorry, there are new French data coming in that @kachergis has been working
on.

**Comment by alvinwmtan** [comment]
Closed in favour of #298
